jdd.js

```diff
@@ -604,24 +604,19 @@ var jdd = {
                 '</span></div>';
             };
 
-            var lines = $(pre).text().split('\n');
+            var lines = pre.textContent.split('\n');
             lines.forEach(addLine);
 
             // Combine it all together
             codeLines += '</div>';
             lineNumbers += '</div>';
 
-            var codeBlockElement = $(
-              '<pre class="codeBlock">' + lineNumbers + codeLines + '</pre>'
-            );
-
-            codeBlockElement.addClass($(pre).attr('class'));
-            codeBlockElement.attr('id', $(pre).attr('id'));
-
-            $(pre).replaceWith(codeBlockElement);
+            var codeBlockElement = '<pre id="'+ pre.id +'" class="codeBlock ' + pre.classList.toString() + '">' + lineNumbers + codeLines + '</pre>';
+            pre.outerHTML = codeBlockElement;
         });
     },
 
+    // TODO: Not being used anywhere
```

Comment:
This is left over from an older implementation where I used to use a text area for the compare results.  I should probably delete it.

Reply:
Shall I delete that in this PR or do that in a separate PR ?

Reply:
It's fine to delete this function as part of this PR, or leave it for another one.

Reply:
I removed as part of this PR